setup.cfg

```diff
@@ -96,6 +96,8 @@ datasets =
     # https://github.com/brianhelba/zipfile-deflate64/issues/19
     zipfile-deflate64>=0.2,<0.3
 docs =
+    # ipywidgets 7+ required by nbsphinx
+    ipywidgets>=7,<8
```

Comment:
I removed this in https://github.com/microsoft/torchgeo/pull/574#discussion_r897595816 because I couldn't remember why it was required and it didn't seem to be needed anymore. Turns out the addition of `black[jupyter]` installed ipywidgets behind the scenes, so that's why the tests didn't fail. Splitting things up into separate files uncovered https://github.com/spatialaudio/nbsphinx/issues/24#issuecomment-1159529577 which explains why I added this in the first place.